client/context/context.go

```diff
@@ -0,0 +1,113 @@
+package context
+
+import (
+	"io"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/wire"
+	"github.com/cosmos/cosmos-sdk/x/auth"
+	"github.com/spf13/viper"
+	rpcclient "github.com/tendermint/tendermint/rpc/client"
+)
+
+const ctxAccStoreName = "acc"
```

Comment:
Hmm should this be a constant? Why not a `QueryContext` field in case we want to change it?

Reply:
In fact we have `QueryContext.AccountStore`

Reply:
Yes, indeed it is already a field in the `QueryContext`, however:
1. There is no path (from what I can tell) where this is actually configurable or desirable (i.e. everywhere in the code uses `acc`)
2. Since there is no CLI flag for it, a user can just do `queryCtx.WithAccountStore(...)`.

I'm happy to revert it to being hard-coded.

Reply:
looks fine for now.